pkg/apiserver/registry/system/controllerinfo/rest_test.go

```diff
@@ -23,13 +23,15 @@ import (
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
+	clientset "k8s.io/client-go/kubernetes"
 
 	"antrea.io/antrea/pkg/apis/crd/v1beta1"
 )
 
 type fakeControllerQuerier struct{}
 
 func (q *fakeControllerQuerier) GetControllerInfo(info *v1beta1.AntreaControllerInfo, partial bool) {}
+func (q *fakeControllerQuerier) GetK8sClient() clientset.Interface                                  { return nil }
```

Comment:
An empty line is desired between functions, and the weird alignment can be avoided with it.